src/coreclr/System.Private.CoreLib/src/System/Array.CoreCLR.cs

```diff
@@ -381,8 +381,52 @@ private unsafe bool IsValueOfElementType(object value)
         // if this is an array of value classes and that value class has a default constructor
         // then this calls this default constructor on every element in the value class array.
         // otherwise this is a no-op.  Generally this method is called automatically by the compiler
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        public extern void Initialize();
+        public unsafe void Initialize()
+        {
+            MethodTable* pArrayMT = RuntimeHelpers.GetMethodTable(this);
+            var thElem = pArrayMT->GetArrayElementTypeHandle();
```

Comment:
```suggestion
            TypeHandle thElem = pArrayMT->GetArrayElementTypeHandle();
```
Style nit - https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

Similar to other vars